probedata.go

```diff
@@ -24,8 +24,20 @@ const (
 
 // ProbeData is the root json data structure returned by an ffprobe.
 type ProbeData struct {
-	Streams []*Stream `json:"streams"`
-	Format  *Format   `json:"format"`
+	Streams  []*Stream   `json:"streams"`
+	Format   *Format     `json:"format"`
+	Chapters []*Chapters `json:"chapters"`
```

Comment:
This would need the `-show_chapters` flag to be present in the ffprobe command, correct? If so, please add it.